views/meetingComplete.ejs

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Thanks for using Vonage Free Conferencing!</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="content-type" content="text/html;charset=UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <link rel="stylesheet" href="/css/completeMeeting.opentok.css" />
+    <link rel="stylesheet" href="css/buttons.css" />
+    <link href="/css/fonts.css" rel="stylesheet">
+    <% if (enableFeedback && hotjarId && hotjarVersion ) { %>
+      <script>
+          (function(h,o,t,j,a,r){
+           h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+           h._hjSettings={hjid: <%=hotjarId%>, hjsv: <%=hotjarVersion%>};
+           a=o.getElementsByTagName('head')[0];
+           r=o.createElement('script');r.async=1;
+           r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+           a.appendChild(r);
+       })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+       </script>
+        <% } %>
+  </head>
+  <body>
+    <section class="completeMeeting">
+        <header>
+            <img src="images/vonage-logo-no-text.svg" alt="vonage transparent logo">
+              <p>Thanks for using<br>Vonage Free Conferencing!</p>
+          </header>
+        <div class="completeMeeting-box-container">
+          <div class="completeMeeting-box">
+            <h2>Built on the Vonage Video API</h2>
+            <p>
+             Did you know that Vonage Free Conferencing is open source and built on the top of the Vonage Video API? 
+            <a  href="https://github.com/opentok/opentok-rtc">
+             Check out the GitHub repo.
+            </a>
+          </p>
+            <p>
+              Have a killer idea for how we can make this app even better? Sign up for a Vonage Video API account and start hacking today!
```

Comment:
"Sign up for a Vonage Video API account" should be a link (to https://tokbox.com/account/user/signup).